app.py

```diff
@@ -21,10 +21,10 @@ def predict():
     #predict = prediction[:, 1]

```

Comment:
GPT summary of [1fcf8a](https://github.com/SampatSuthar/ml-demo/blob/f3fc2c559b5eac826cd89d51e16fef43ebaff446/app.py#1fcf8aed8558b2618a97636c2f6d6c3587c2a6ee) - [1912d4](https://github.com/SampatSuthar/ml-demo/blob/8d2c6859e280f41855b84ac5379522d932133d44/app.py#1912d43f95c066b874d8646e52a77052c42facc9):
* Removed the import of the openai library and the setting of the API key
* Changed the model name from "text-davinci-003" to "davinci-003"
* Added a prompt for the completion of the OpenAI API
* Added a variable to store the generated text from the OpenAI API